src/api.py

```diff
@@ -25,7 +27,32 @@ def home():
         return 'WELCOME TO CABINET'
 
 
-    @app.route('/blob', methods=['GET', 'POST', 'PUT'])
+    @app.route('/hosts', methods=['GET'])
+    def hosts():
+        with open('config/config.yaml','r') as file:
+            config_dict = yaml.safe_load(file) 
+        hosts = list(config_dict['save_hosts'].keys())
+        return Response(body={'hosts':hosts}).json() 
+        
+
+    @app.route('/paths', methods=['GET'])
+    def paths():
+        try:
+            conn, cur = db_connect(env=env)
+            new_blob_unsaved = PathsSchema(request.args.to_dict())
```

Comment:
you need to pass in the blob type metadata here too, because you need to guarantee that the user has the correct data before they save their blob.

Reply:
Done